core/src/test/java/org/eclipse/hono/connection/ConnectionFactoryImplTest.java

```diff
@@ -73,6 +77,37 @@ public void testConnectInvokesHandlerOnfailureToConnect(final TestContext ctx) {
         handlerInvocation.await(2000);
     }
 
+    /**
+     * Verifies that the given result handler is invoked if a connection gets closed after SASL auth was successful and
+     * AMQP open frame was sent by client, but no AMQP open frame from server was received yet.
+     */
+    @SuppressWarnings("unchecked")
+    @Test
+    public void testConnectInvokesHandlerOnDisconnectAfterSendingOpenFrame(final TestContext ctx) {
+        // GIVEN a factory configured to connect to a server (with mocked connection)
+        ConnectionFactoryImpl factory = new ConnectionFactoryImpl(vertx, props);
+        ProtonClient protonClientMock = mock(ProtonClient.class);
```

Comment:
can you please make the declared variables final

Reply:
done :)